sdk/monitor/monitor-opentelemetry-exporter/src/export/constants.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+export const STATSBEAT_LANGUAGE = "node";
+
+export const MAX_STATSBEAT_FAILURES = 3;
+
+export const redirectStatusCodes = [
+  307, // Temporary redirect
+  308 // Permanent redirect
+];
+
+export const retryableStatusCodes = [
+  206, // Partial success
```

Comment:
Do we have these somewhere else?, we should use these for actual retrying in the exporter

Reply:
Yeah, looks like they already exist in the breezeUtils, I'll clean this up!